lib/internal/repl.js

```diff
@@ -59,7 +61,17 @@ function createRepl(env, cb) {
 }
 
 function setupHistory(repl, historyPath, ready) {
-  const fs = require('fs');
+  // default to using <homedir>/tmp/node_history_file
```

Comment:
Looks like it assuming this is the correct bit of code: https://github.com/npm/npm/blob/064d62c2f251cb5e9328f63228f711844b7a1787/lib/config/defaults.js#L208